src/_includes/macros/expandable.html

```diff
@@ -8,26 +8,45 @@
 
    Builds Expandable markup when given:
 
-   items                     : An array of expandable items.
+   items                      : An array of expandable items.
 
-   header_key                : A string used to denote the header dictionary key.
+   header_key                 : A string used to denote the header dictionary key.
 
-   content_key               : A string used to denote the content dictionary key.
+   content_key                : A string used to denote the content dictionary key.
 
-   options (optional)        : An object used to customize
-                               the expandable markup.
+   options (optional)         : An object used to customize
+                                the expandable markup.
 
-   options.title             : Title to use for expandable markup.
+   options.title              : Title to use for expandable markup.
 
-   options.additional_classes: Extra classes added to the expandable markup.
+   options.additional_classes : Extra classes added to the expandable markup.
+
+   options.faq_description    : Description used for FAQ markup.
+
+   options.faq_title          : Title to used for FAQ markup.
 
    ========================================================================== #}
 {% macro render(items=[], header_key='', content_key='', options={}) -%}
-    {% set title = options.title | default('&nbsp;') | safe %}
+    {% if options.faq_title or options.faq_description %}
+    <div class="u-js-only">
+        {% if options.faq_title %}
+        <h1 class="h2">
```

Comment:
@sebworks You mean store the hardcoded title values? If we ever wanted to do proper internationalization we'd probably want that, so seems fine with me. It'd probably make sense to do it in one sweep across the project though in another PR.